src/subcommand/wallet/inscribe.rs

```diff
@@ -278,7 +382,7 @@ impl Inscribe {
     );
 
     let witness = sighash_cache
-      .witness_mut(0)
+      .witness_mut(commit_input_offset)
       .expect("getting mutable witness reference should work");
     witness.push(signature.as_ref());
```

Comment:
When inscribing with a `parent`, do we need to encode the `signature` with the proper SigHashType before adding to the witness?
```
let encoded_sig = SchnorrSig { sig: signature, hash_ty: SchnorrSighashType::AllPlusAnyoneCanPay };
witness.push(encoded_sig.serialize());
```